cosmicfishpie/configs/config.py

```diff
@@ -377,14 +388,22 @@ def survey_equivalence(surv_str):
     surveyNameSpectro = settings.get("survey_name_spectro")
     if "Euclid" in surveyName:
         if surveyName == "Euclid" and surveyNamePhoto == "" and surveyNameSpectro == "":
```

Comment:
I agree with you. What is needed is a global_default.yaml that can always fill-in everything that is not set.
However, this probably requires a different MR. In here, I think we can safely assume that the default_specifications folder will always be found (it is installed with CF). In my new commit I have added some defaults to these two variables, so that they always fall back to ISTF pessimistic. Thanks for the catch!

Reply:
Yes that would be great!